src/scss/components/dropdown/dropdown.md

```diff
@@ -1,29 +1,9 @@
 # Dropdown
 
 ```html
-<div class="has-dropdown">
-  <button class="Btn" aria-controls="dropdownBottomLeft" aria-expanded="true">Call to action</button>
-	<ul id="dropdownBottomLeft" class="Dropdown">
-	  <li class="Dropdown-item">
-	    Simple label
-	  </li>
-	  <li>
-	    <a href="#" class="Dropdown-item is-active">
-	      Linked label
-	    </a>
-	  </li>
-	</ul>
-</div>
-```
-
-## Positions
-
-### Bottom-centered
-
-```html
-<div class="has-dropdown">
-  <button class="Btn" aria-controls="dropdownBottomCentered" aria-expanded="true">Call to action</button>
-  <ul id="dropdownBottomCentered" class="Dropdown Dropdown--bc">
+<div class="Dropdown" data-tapestry="dropdown">
+  <button class="Dropdown-toggle Btn" aria-controls="dropdownBottomLeft" aria-expanded="true">Call to action</button>
```

Comment:
Could we use something else to replace `dropdownBottomLeft`? Seems confusing to me.